code/game/gamemodes/clock_cult/clock_mobs.dm

```diff
@@ -172,7 +172,8 @@
 /mob/living/simple_animal/hostile/clockwork_marauder/proc/adjust_fatigue(amount) //Adds or removes the given amount of fatigue
 	if(!ratvar_awakens)
 		fatigue = max(0, min(fatigue + amount, fatigue_recall_threshold))
-	return 1
+	Life() //Immediately runs a life tick to check for recalling
```

Comment:
Put this under the if(), no need to check if ratvar's up
Also add an else setting amount to 0, so if ratvar's up it properly returns nothing